### DISCUSSION TITLE
when use skywalking-client-js and pdfh5 ，Some strange questions

### DISCUSSION THREAD

**Discussion by baixjian** [apache/skywalking#10499] [Q&A]
When not using sw, it can display pdf file,
Then I `ClientMonitor.register` sw and there's a cross-domain problem.
<img width="1670" alt="image" src="https://user-images.githubusercontent.com/109275788/223637743-6b35a1f4-5a04-477f-9abc-071ccbcd2b0d.png">
I have no idea what's wrong. Can someone help me。
Your help is greatly appreciated。

There's a minimal recurrence：
[demo](https://github.com/baixjian/sw-demo)

**Comment by Fine0830** [comment]
I think it was caused by tracing, you should use `noTraceOrigins` to exclude pdfh5 requests.
